doonut-app-external-api/src/main/kotlin/com/doonutmate/oauth/kakao/service/KakaoOauthLoginService.kt

```diff
@@ -0,0 +1,35 @@
+package com.doonutmate.oauth.kakao.service
+
+import com.doonutmate.doonut.member.model.Member
+import com.doonutmate.doonut.member.model.OauthType
+import com.doonutmate.doonut.member.service.MemberBusinessService
+import com.doonutmate.oauth.kakao.dto.KakaoTokenRequest
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+@Transactional
+class KakaoOauthLoginService(
+    private val kakaoAccessClientLoginService: KakaoAccessClientLoginService,
+    private val memberBusinessService: MemberBusinessService,
+) {
+
+    fun kakaoLoginPreventDuplication(tokenRequest: KakaoTokenRequest, oauthType: OauthType): Member? {
```

Comment:
반환하는 Member 객체가 null이 아님이 보장되니까 ?을 제거해도 좋을 것 같아요.  
```suggestion
    fun kakaoLoginPreventDuplication(tokenRequest: KakaoTokenRequest, oauthType: OauthType): Member {
```

Reply:
메서드명에 PreventDuplication을 제거해도 좋을 것 같아요.